src/release/packageApi.js

```diff
@@ -153,7 +163,10 @@ export default function packageApiFactory(params = {}, data) {
          * @returns {Promise<void>}
          */
         async monorepoPublish() {
-            await this.npmPackage.lernaPublish();
+            const publishPackage = await this.beforePublish();
+            if (publishPackage)     {
+                return this.npmPackage.lernaPublish();
+            }
```

Comment:
You could add a `log.doing()` to make this function more closely match `publish()`? (I didn't check if output will be duplicated - no need to add it if that is the case.)

Reply:
Ok. It will also print out `npx lerna publish from-package --yes` and its output, so details would be clear anyway.